docs/content/4.sdk/2.getting-started/4.logger.md

```diff
@@ -0,0 +1,169 @@
+# Logger
+
+The Logger provides a logging utility for Alokai Storefront projects. It allows you to log messages at various levels and ability to provide additional context. 
+
+It is created to provide a unified way of logging messages across the project along with providing additional data out of the box.
+
+## Installation
+
+If the Logger is already included in the Alokai Storefront, you can skip this step.
+
+The Logger is provided by the SDK and framework specific packages.
+
+In order to install the Logger, you need to update following packages to at least the following versions:
+* `@vue-storefront/sdk` to `3.3.0`
+* `@vue-storefront/nuxt` to `6.2.0`
+* `@vue-storefront/next` to `4.3.0`
+  
+After updating the packages, you need to provide the Logger module to the SDK config.
+
+::tabs{:titles='["Next.js", "Nuxt"]' class="mt-8"}
+
+#tab-1
+
+```ts
+// apps/storefront-unified-nextjs/sdk/config.ts
+
+// ...
+export function getSdkConfig() {
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => ({ // [!code --]
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, loggerModule, middlewareModule }) => ({ // [!code ++]
+    logger: buildModule(loggerModule), // [!code ++]
+    // ...
+  }));
+}
+```
+
+#tab-2
+
+```ts
+// apps/storefront-unified-nuxt/sdk/config.ts
+
+// ...
+export default defineSdkConfig(({ buildModule, middlewareModule, getRequestHeaders, config }) => ({ // [!code --]
+export default defineSdkConfig(({ buildModule, loggerModule, middlewareModule, getRequestHeaders, config }) => ({ // [!code ++]
+  logger: buildModule(loggerModule), // [!code ++]
+  // ...
+}));
+```
+::
+
+
+## Configuration
+
+The default configuration is already provided, but you can customize it by providing the following options:
+
+* `level` - the minimum level of the message to be logged. The default value is `info`. Possible values are:
+  * `emergency` 
+  * `alert`
+  * `critical`
+  * `error`
+  * `warning`
+  * `notice`
+  * `info`
+  * `debug`
+* `includeStackTrace` - a boolean value that determines if the stack trace should be included in the log message. The default value is `true`.
+* `handler` - a custom handler that will be called when the message is logged.
+
+To provide the configuration for the Logger, you need to update the SDK config with the Logger module configuration by providing an object with options to the `buildModule` function, e.g.:
+```ts
+  // ...
+  logger: buildModule(loggerModule, {
+    level: 'debug',
+    includeStackTrace: false,
+    handler: myCustomHandler,
```

Comment:
Providing `handler` makes other configuration properties redundant. I think we should either present other configuration properties or only handler property here.

Reply:
My idea was to put all available options here but yeah, this might be a better idea. Especially that all options are listed above and handler has its own section below.